packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -456,7 +457,7 @@ export default class Meetings extends WebexPlugin {
 
     /**
      * Create a meeting.
-     * @param {string} destination - sipURL, spaceId, phonenumber, meeting link, or locus object}
+     * @param {string} destination - sipURL, spaceId, personId, phonenumber, meeting link, or locus object
```

Comment:
should add hydra room id

Reply:
also we should change spaceId to spaceUrl

Reply:
I've added roomId to the JSDoc as mentioned.